forward.go

```diff
@@ -25,18 +28,22 @@ var (
 	_ llb.SecretOption = (*Chmod)(nil)
 )
 
+// WithChmod creates a Chmod option with the provided os.FileMode
```

Comment:
nit: The other doc comments end with `.`, these don't even though they're complete sentences.